### PR TITLE
CacheStorage API requires secure context in Chrome/Safari/Firefox

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Continuation off of #5169 due to lack of response from the original author.

### CODE REVIEW

File: api/CacheStorage.json [hunk 1]
```diff
@@ -321,6 +321,51 @@
             "deprecated": false
           }
         }
+      },
+      "secure_context_required": {
+        "__compat": {
+          "description": "Secure context required",
+          "support": {
+            "chrome": {
+              "version_added": true
```
Comment (jpmedley) [inline api/CacheStorage.json]: This landed in 65: https://storage.googleapis.com/chromium-find-releases-static/c32.html#c3231dc7d9d57c5dcf5dc0f0a6bb4af48aba85a4

File: api/CacheStorage.json [hunk 2]
```diff
@@ -321,6 +321,51 @@
...
+            },
+            "chrome_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/CacheStorage.json]: 65

File: api/CacheStorage.json [hunk 3]
```diff
@@ -321,6 +321,51 @@
...
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
```
Comment (jpmedley) [inline api/CacheStorage.json]: 65

File: api/CacheStorage.json [hunk 4]
```diff
@@ -321,6 +321,51 @@
...
+              "version_added": "65"
+            },
+            "chrome_android": {
+              "version_added": "65"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
```
Comment (ddbeck) [inline api/CacheStorage.json]: I tested this one

```suggestion
              "version_added": "44"
```

File: api/CacheStorage.json [hunk 5]
```diff
@@ -321,6 +321,51 @@
...
+            },
+            "firefox_android": {
+              "version_added": null
```
Comment (ddbeck) [inline api/CacheStorage.json]:
```suggestion
              "version_added": "44"
```

**Comment by vinyldarkscratch** [comment]
Haha, not a problem!  After all, now we have some more data put right into the initial PR, right?  :wink:

**Pull request merged**